General/CJKCharacter.md

```diff
@@ -0,0 +1,31 @@
+# Using CJK (Chinese, Japanese, and Korean) characters
+
+Pharo's default font settings for code (Source Code Pro) and UI (Source Sans Pro) are unable to handle [CJK characters](https://en.wikipedia.org/wiki/CJK_characters). 
+
+![](CJKCharacter_Screenshot_1.png)
```

Comment:
You should put an alt text to this picture in case it becomes unavailable for some reason.

Reply:
And also for accessibility (blind people, etc)